gravitee-apim-console-webui/src/management/api-score/api-score-dashboard/api-score-dashboard.component.html

```diff
@@ -43,7 +43,7 @@ <h1 class="page-title">API Score</h1>
       <mat-card class="card" data-testid="overview-score">
         <mat-card-header>
           <mat-card-title class="card__title">
-            @if (!overviewData || overviewData.score === undefined) {
+            @if (!overviewData || overviewData.score === undefined || overviewData.score === null) {
```

Comment:
shouldn't we use `== nul`l to check both `undefined` and `null` 🤔https://stackoverflow.com/questions/2559318/how-to-check-for-an-undefined-or-null-variable-in-javascript

Reply:
yes, it is one of the way, but not rule :-)